src/cfml/system/modules_app/system-commands/commands/jq.cfc

```diff
@@ -57,30 +66,69 @@ component {
 	function run(
 		required string inputOrFile,
 		string query=''
-		)  {
+	)  {
+
+		var folderName = tempDir & '/' & 'temp#createUUID()#';
+		var fullPath = folderName & '/' & createUUID() & '.json';
+
+		directoryCreate( folderName, true, true );
```

Comment:
@scottsteinbeck Don't create the temp dir unless we're actually doing a download.  Also, all temp folders and files need to be deleted inside a "finally" block to not fill up the temp dir.